i18n/en.json

```diff
@@ -50,6 +50,7 @@
 	"maps-coordinates-par-location": "The coordinates you want to format.",
 	"maps-coordinates-par-format": "The target format for the coordinates.",
 	"maps-coordinates-par-directional": "Indicates if the coordinates should be outputted directional or not.",
+	"maps-coordinates-par-scrollwheelzoom": "Indicates if mouse scrolling should be enabled or not.",
```

Comment:
Let me know if that sentence is good enough ;)

Reply:
Yeah but the key is now

'maps-par-scrollwheelzoom'

This is not a parameter for the coordinates parser function :)